core/src/test/java/org/apache/iceberg/TestPartitioning.java

```diff
@@ -140,9 +140,10 @@ public void testPartitionTypeWithAddingBackSamePartitionFieldInV1Table() {
         .addField("data")
         .commit();
 
-    // in v1, we use void transforms instead of dropping partition fields
+    // in v1, we use void transforms instead of dropping partition fields.
+    // We restored the void transforms with the original dropped partition fields.
```

Comment:
I'm not sure line 144 here makes sense to me, I think this is fine without the comment. In fact I think the expected type makes more sense now than it previously did.

Reply:
OK, remove it.